src/main/java/com/atlauncher/Launcher.java

```diff
@@ -376,8 +374,8 @@ public void reloadLauncherData() {
             checkForExternalPackUpdates();
 
             ConfigManager.loadConfig(); // Load the config
-            NewsManager.loadNews(); // Load the news
-            reloadNewsPanel(); // Reload news panel
+            NewsManager.loadFileNews(); // Load the news
+            NewsManager.loadNetworkNews(); // Reload news panel
```

Comment:
Seems this would load news from both sources if enabled right? Shouldn't it just be a single `loadNews` call like before that does if/else on the config toggle?

Reply:
That was not what it did prior.

`NewsManager.loadNews()` always reads from the file system into `Data.NEWS`

`reloadNewsPanel()` ends up at `NewsViewModel.reload()` which then uses a config check to decide what to load.

But I see what you want, and will implement the change.